src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -434,6 +457,12 @@ private void refreshTripVehicle() throws IOException, JSONException {
 						VehiclePosition.Builder vehiclePosition = VehiclePosition.newBuilder();
 						vehiclePosition.setPosition(position);
 						vehiclePosition.setTrip(tripDescriptor);
+
+						if (info.APCPercentage <= 0) vehiclePosition.setOccupancyStatus( VehiclePosition.OccupancyStatus.EMPTY );

```

Comment:
See https://github.com/CUTR-at-USF/bullrunner-gtfs-realtime-generator/issues/4#issuecomment-76208281 for some notes on cutoff values.